lib/controllers/syncController.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:get/get.dart';
+import 'package:knocky/helpers/api.dart';
+
+class SyncController extends GetxController {
+  final isFetching = false.obs;
+  final subscriptionNotificationCount = 0.obs;
+
+  static final SyncController _singleton = SyncController._internal();

```

Comment:
The thought here is that we might want to consume sync-data elsewhere later and populate the same sync-data to subscribers, so I thought a singleton makes sense.